google-cloud-video_intelligence/docs/toc.json

```diff
@@ -6,6 +6,21 @@
       "type": "google/cloud/videointelligence",
       "patterns": ["\/videointelligence", "google\/protobuf", "google\/rpc"],
       "nav": [
+        {
```

Comment:
@quartzmo Won't this also have to be added to the `google-cloud/docs/toc.json`?

Reply:
@blowmage Good catch. We need to always remember to look outside the scope of the package directory with these GAPIC updates. 

@geigerj  [This section](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud/docs/toc.json#L587-L618) of `google-cloud/docs/toc.json` should also be updated.

Reply:
Updated.

Reply:
@blowmage This is fixed. Anything else, or is this ready to merge?